src/modules/Player/Components/SkipButton/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+type Props = {
+  children: React.ReactNode;
+  trackNumber: number;
+  tracks: number;
+  setTrackNumber: React.Dispatch<React.SetStateAction<number>>;
+};
+
+const SkipButton: React.FC<Props> = React.memo(({ children, trackNumber, tracks, setTrackNumber }) => {
+  const NextTrack = () => {
+    if (trackNumber == tracks - 1) {
+      return trackNumber
+    }
+    else {
+      setTrackNumber(trackNumber + 1)
+    }
+  };
+  const PreviosTrack = () => {
+    if (trackNumber == 0) {
+      return trackNumber
+    }
+    else {
+      setTrackNumber(trackNumber - 1)
+    }
+  };
```

Comment:
Идея хорошая, но 3 момента:
```suggestion
 if (!condition) {
do smth
return
}

if correct - do smth
  ```
1. проверяй отрицание всегда первым

1. Используй строгое сравнение (===, а не ==)


1. Дальше, проверяешь, что если трек равен длине массива, то есть он последний, значит устанавливаешь значение на 0. То есть будет первый трек. В ином же случае увеличиваешь текущий трек на 1